http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/KubernetesClusterComponent.scala

```diff
@@ -246,14 +210,30 @@ object kubernetesClusterQuery extends TableQuery(new KubernetesClusterTable(_))
 
 }
 
-case class SaveKubernetesCluster(
-  googleProject: GoogleProject,
-  clusterName: KubernetesClusterName,
-  location: Location,
-  status: KubernetesClusterStatus,
-  serviceAccount: WorkbenchEmail,
-  samResourceId: KubernetesClusterSamResourceId,
-  auditInfo: AuditInfo,
-  labels: LabelMap,
-  initialNodepool: Nodepool
-) //the clusterId specified here isn't used, and will be replaced by the id of cluster saved beforehand
+case class KubernetesAppCreationException(message: String) extends Exception
+
+case class SaveKubernetesCluster(googleProject: GoogleProject,
+                                 clusterName: KubernetesClusterName,
+                                 location: Location,
+                                 status: KubernetesClusterStatus,
+                                 serviceAccount: WorkbenchEmail,
+                                 auditInfo: AuditInfo,
+                                 dummyNodepool: Nodepool) {
```

Comment:
Yeah, its used here https://github.com/DataBiosphere/leonardo/pull/1443/files#diff-aa9679f7f59adb5590ab00dfff5fe0e6R187. 

Its because a cluster needs a nodepool to be created in kubernetes, so I wanted to enforce it in the ADT

Reply:
The term GCP uses for the nodepool that needs to be created as part of the cluster is `default nodepool`. How would you feel about renaming `dummy` as `default` so it's one less thing to know.

Reply:
I originally had it as default, but we've been calling it 'dummy' as a team, so I was thinking it could be more approachable this way. I'm fine with changing it back